packages/@aws-cdk/aws-cloudwatch/test/cross-environment.test.ts

```diff
@@ -177,6 +177,63 @@ describe('cross environment', () => {
         ],
       });
     });
+
+    test('math expression can render in a different account', () => {
+      // GIVEN
+      const b = new Metric({
+        namespace: 'Test',
+        metricName: 'ACount',
+        account: '1234',
+      });
+
+      const c = new MathExpression({
+        expression: 'a + b',
+        usingMetrics: { a: a.attachTo(stack3), b },
+        period: Duration.minutes(1),
+      });
+
+      new Alarm(stack1, 'Alarm', {
+        threshold: 1,
+        evaluationPeriods: 1,
+        metric: c,
+      });
+
+      // THEN
+      Template.fromStack(stack1).hasResourceProperties('AWS::CloudWatch::Alarm', {
+        Metrics: [
+          {
+            Expression: 'a + b',
```

Comment:
This might be out of scope for this change, but shouldn't this expression have `ReturnData: true`? See https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cloudwatch-alarm-metricdataquery.html#cfn-cloudwatch-alarm-metricdataquery-returndata 

Did you try to deploy this with a sample app or an integ test to see if it actually works?

Reply:
Only 1 metric can have `ReturnData = true` so it makes sense to default to `false` for all of these. I am inclined to believe that the comment in the code is incorrect. I have traced it to [this](https://github.com/aws/aws-cdk/pull/5582/files) commit and there doesn't seem to be any discussion to clear up why this comment doesn't make sense. I have updated the comment to say default is `false`.